cdap-master/pom.xml

```diff
@@ -491,6 +491,9 @@
                         <exclude name="${additional.artifacts.exclude.pattern}"/>
                         <exclude name="**/target/*-sources.jar"/>
                         <exclude name="**/target/*-javadoc.jar"/>
+                        <!-- Wrangler excludes --> 
+                        <exclude name="**/target/wrangler-core*"/>
+                        <exclude name="**/target/wrangler-test*"/>
                         <!--We don't want to package the following plugins with CDAP-->
                         <exclude name="**/target/solrsearch-plugins*"/>
```

Comment:
wait... we have two different strategies for not packaging plugins in CDAP?:
1) avoid putting it in hydrator-plugins repo
2) excluding it in the pom here